### ISSUE TITLE
Compatibility with Grafana version 8?

### ISSUE THREAD

**Issue by romansoft** [vsergeyev/loudml-grafana-app#44]
I'm trying to use this plugin with Grafana 8, but the data source type is not being listed (after approving unsigned plugins). Would it be due to incompatibility with this new version?

**Comment by allamiro** [comment]
yes I m getting also the same error using grafana 8 Missing signature

<img width="1263" alt="Screen Shot 2021-09-14 at 1 17 33 AM" src="https://user-images.githubusercontent.com/279790/133163721-454f12bf-d085-402e-8018-1b0eab16c6cf.png">

https://grafana.com/docs/grafana/latest/developers/plugins/sign-a-plugin/
